src/commands/helm-install-cli.yml

```diff
@@ -1,3 +1,5 @@
+---
+
 description: Install Helm local client.
 
 steps:
```

Comment:
🐼 

Maybe worth to update to Helm version 3.12.2 (currently locally, as well as in Terraform provider)
https://github.com/hashicorp/terraform-provider-helm/blob/main/go.mod#L13C2-L13C25

Reply:
Done